.github/workflows/deploy_to_s3.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy to AWS S3
+
+on:
+  push:
+    branches:
+      - main 
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository code
+      uses: actions/checkout@v2 
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }} 
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }} 
+        aws-region: eu-west-2 
+
+    - name: Deploy to S3
+      run: |
+        aws s3 sync ./weather-app s3://cammo1309-weather-app1 --delete 
```

Comment:
Lovely ! 🎉 

I think you'll be able to replace the `aws s3 sync` call when you wire in some terraform [and let that handle the syncing with s3](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_object).

Reply:
Oooh interesting, thank you!